styles/colors-silicon-day.less

```diff
@@ -52,7 +52,7 @@
 @agate-darker-gray: @silicon-dark;
 @agate-dark-gray:   @silicon-dark-two;
 @agate-gray:        #8a8a8a;
-@agate-light-gray:  #ededed;
+@agate-light-gray:  #cccccc;
```

Comment:
How about `@agate-very-light-gray`?
https://www.htmlcsscolor.com/hex/CCCCCC

`#D0D0D0` could be `light-gray`, I think. This color could be used later. Just in case.

Reply:
good idea. i just didn't know a good name for the new value

Reply:
now I see that in color-base.less we already have: @agate-light-gray:  #ccc;
Would it be a better idea for silicon to just inherit this value?

Reply:
Oh I got it. Let's inherit it first in the PR, then rename it later when needing D0D0D0 :)